pollux/core/src/main/scala/org/hyperledger/identus/pollux/core/model/PresentationRecord.scala

```diff
@@ -34,6 +35,7 @@ final case class PresentationRecord(
     metaRetries: Int,
     metaNextRetry: Option[Instant],
     metaLastFailure: Option[Failure],
+    walletId: WalletId,
```

Comment:
I can't find the Schema migration script that add this field on the DB side. Can you point me to it?

Reply:
I believe the column is already there

Reply:
Had a call with Fabio. It already there